src/main/java/mate/academy/library/dto/category/CategoryDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.library.dto.category;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class CategoryDto {
+    private Long id;
+    @NotNull

```

Comment:
```suggestion
    @NotBlank

```
it's better for String